README.md

```diff
@@ -80,6 +80,29 @@ civicClient.exchangeCode(jwtToken)
     });
 ```
 
+### Ephemeral Token
```

Comment:
Would it be possible to add a use case description here and show how it would be used on the front end? I think it would help anyone coming here to understand when to use this token?